swift/apple/FirezoneKit/Sources/FirezoneKit/Models/Settings.swift

```diff
@@ -35,19 +37,29 @@ struct Settings: Equatable {
         apiURL: providerConfiguration[TunnelManagerKeys.apiURL]
           ?? Settings.defaultValue.apiURL,
         logFilter: providerConfiguration[TunnelManagerKeys.logFilter]
-          ?? Settings.defaultValue.logFilter
+          ?? Settings.defaultValue.logFilter,
+        disabledResources: getDisabledResources(disabledResources: providerConfiguration[TunnelManagerKeys.disabledResoruces])
```

Comment:
```suggestion
        disabledResources: getDisabledResources(disabledResources: providerConfiguration[TunnelManagerKeys.disabledResources])
```

Since this is stored next to the log filter and all, that means resetting the settings also re-enables all Resources, right? Just checking I know where it's stored.

Reply:
Yeah, I guess it's fine? maybe we can add some docs, we can also change it so that the "Reset to defaults" doesn't reset this value it's quite trivial but it might not be worth it